File: Programming Fundamentals/04.DataTypesAndVariablesExercises/13.VowelOrDigit/VowelOrDigit.cs
using System;

namespace _13.VowelOrDigit
{
    public class VowelOrDigit
    {
        public static void Main(string[] args)
        {
            string symbol = Console.ReadLine().ToLower();

            int value;

            if (int.TryParse(symbol, out value))
            {
                Console.WriteLine("digit");
            }
            else if (symbol == "a" || symbol == "e" || symbol == "i" || symbol == "o" || symbol == "u" || symbol == "y")
            {
                Console.WriteLine("vowel");
            }
            else
            {
                Console.WriteLine("other");
            }

        }
    }
}

File: Software Technologies/10. Java Syntax Basic Exercises/src/com/company/VowelOrDigit.java
package com.company;

import java.util.Scanner;

public class VowelOrDigit {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        char input = scanner.next().charAt(0);

        if (Character.isDigit(input)){
            System.out.print("digit");
        } else if (isVowel(input)) {
            System.out.print("vowel");
        } else {
            System.out.print("other");
        }
    }

    public static boolean isVowel(char ch) {
        return "AEYIOUaeyiou".indexOf(ch) != -1;
    }
}
